### PR TITLE
custom-component: percentage component

### PR DESCRIPTION
**Pull request by OlegRovenskyi**

### CODE REVIEW

File: src/app/shared/form-components/percentage/percentage.component.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      useExisting: forwardRef(() => PercentageComponent),
+      multi: true
+    }
+  ]
+})
+export class PercentageComponent implements OnInit {
+  @Input()
+  public percentage: string;
+  @Output()
+  public setOrderPercentage: EventEmitter<string> = new EventEmitter<string>();
```
Comment (korniychuk) [inline src/app/shared/form-components/percentage/percentage.component.ts]: Let's avoid this Input & Output. forms functionality should be enough.

File: src/app/shared/form-components/percentage/percentage.component.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<mat-form-field>
+  <input type="number" matInput placeholder="Percent" [value]="value">
```
[the remark this answers is not in the document]
Comment (korniychuk) [reply src/app/shared/form-components/percentage/percentage.component.html]: Update: it doesn't work. I checked

File: src/app/shared/form-components/percentage/percentage.component.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+  public percentRange: string[] = ['10', '20', '25', '50', '75', '80', '100'];
+
+  private _value: string;
```
[the remark this answers is not in the document]
Comment (OlegRovenskyi) [reply src/app/shared/form-components/percentage/percentage.component.ts]: why?

[the remark this answers is not in the document]
Comment (OlegRovenskyi) [reply src/app/shared/form-components/percentage/percentage.component.ts]: if all values is string.

File: src/app/shared/form-components/percentage/percentage.component.ts [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    }
+  ]
+})
+export class PercentageComponent implements OnInit {
+  @Input()
+  public percentage: string;
+  @Output()
+  public setOrderPercentage: EventEmitter<string> = new EventEmitter<string>();
+
+  public percentRange: string[] = ['10', '20', '25', '50', '75', '80', '100'];
```
[the remark this answers is not in the document]
Comment (OlegRovenskyi) [reply src/app/shared/form-components/percentage/percentage.component.ts]: why?

[the remark this answers is not in the document]
Comment (OlegRovenskyi) [reply src/app/shared/form-components/percentage/percentage.component.ts]: if all values is string.

**Pull request merged**